ons_alpha/bulletins/models.py

```diff
@@ -46,7 +46,7 @@ class BulletinPage(BundledPageMixin, RoutablePageMixin, BasePage):
         related_name="+",
     )
     is_accredited = models.BooleanField(default=False)
-    headline_figures = StreamField([("figures", HeadlineFiguresBlock())], blank=True)
+    headline_figures = StreamField([("figures", HeadlineFiguresBlock())], blank=True, max_num=1)
```

Comment:
I actually made this change on the styling branch but makes more sense to do it in a separate commit.